docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -205,6 +205,28 @@ Many methods that take a commitment parameter return an RpcResponse JSON object
 - `context` : An RpcResponseContext JSON structure including a `slot` field at which the operation was evaluated.
 - `value` : The value returned by the operation itself.
 
+#### Parsed Responses
+
+Some methods support an `encoding` parameter, and can return account or
+instruction data in parsed JSON format if `"encoding":"jsonParsed"` is requested
+and the node has a parser for the owning program. Solana nodes currently support
+JSON parsing for the following native and SPL programs:
+
+| Program | Account State | Instructions |
```

Comment:
How about a column for notes on the release versions that support for various states/instructions was added in?

Reply:
Not a bad idea. Can we merge this as-is and then add the version column? I'd rather not dig through github history for all these at the moment.

Reply:
Yup, no problem

Reply:
Version is only useful it's a recent version in beta still so anything stable and older is fine to label as "Stable"

Reply:
Okay, fair enough. Updated.